pages/index.js

```diff
@@ -11,11 +13,18 @@ export default function Home() {
         <H2>save time</H2>
         <H1>Just Practice</H1>
       </HeadlinesWrapper>
-      <Navbar>
+      <StartWrapper>
         <Link href="/timer">
           <Start>Start!</Start>
         </Link>
-      </Navbar>
+      </StartWrapper>
+      <EditWrapperWrapper>
+        <EditWrapper>
```

Comment:
The naming is a bit confusing.

Reply:
you are right. What about `<EditWrapperContainer>` instead?

Reply:
Why exactly do you need two divs? Can't you make the give the EditWrapper all the styling you need and delete the container?
If not, I'm more learning towards "EditingContainer" or something like that